annotation-web/src/main/java/eu/europeana/annotation/web/service/impl/AdminServiceImpl.java

```diff
@@ -140,12 +128,11 @@ public BatchProcessingStatus reindexAnnotationSet(List<String> ids, boolean isOb
 						if (isObjectId) {
 							annotation = getMongoPersistence().findByID(id);
 						} else {
-							annoId = JsonUtils.getIdHelper().parseAnnotationId(id);
-							annotation = getMongoPersistence().find(annoId);
+							annotation = getMongoPersistence().find(Long.parseLong(id));
```

Comment:
the ids are meant to be the annotation URIs, if the method is changed to use the identifiers, it should use a List<long>

Reply:
we should have also the method findByAnnoUri